instance/models/openedx_appserver.py

```diff
@@ -23,6 +23,7 @@
 from django.db import models
 from django.template import loader
 from django.utils.text import slugify
+from django_extensions.db.fields.json import JSONField
```

Comment:
Ah, damn - proper JSON fields would be nicer, and might also fix the fixture serialization issue that @haikuginger was asking about. Unfortunately upgrading PostgreSQL can be a bit tricky.